src/browser-frontend/views/browser/chrome/tabbar.css

```diff
@@ -15,16 +15,17 @@ specific language governing permissions and limitations under the License.
   height: var(--theme-tabbar-height);
   box-sizing: border-box;
   flex: 1;
-  padding: 0 10px;
+  padding: 0 10px 0 40px;
```

Comment:
Any reason why we need both margins and paddings (here and below) to make the tabbar not overflow over window controls? Why not just use one?